packages/palette-docs/content/docs/components/StaticCountdownTimer.mdx

```diff
@@ -6,18 +6,7 @@ name: StaticCountdownTimer
   <StaticCountdownTimer
     action="Respond"
     note="Before it's too late."
-    countdownStart="2019-01-01T12:00:00.000-04:00"
-    countdownEnd="2019-01-14T15:30:00.000-04:00"
-    currentTime="2019-01-05T12:00:30.000-04:00"
-  />
-</Playground>
-
-<Playground>
-  <StaticCountdownTimer
-    action="You forgot to do the thing 😵"
```

Comment:
Why remove this?

Reply:
Just fixed this accidentally pushed it, you're too fast on the trigger :)